lib/nmatrix/monkeys.rb

```diff
@@ -46,7 +46,7 @@ def to_nm(shape = nil, dtype = nil, stype = :dense)
 
     guess_dtype = ->(type) {
       case type
-      when Fixnum   then :int64
+      when Integer  then :int64
```

Comment:
This may present a problem further down the road. Bignum can store more than int64, and this method should probably check for overflow and be able to choose `:object` instead of `:int64`. I think the solution is probably to make `:object` the default dtype for `Integer`.

With that said, I'm not sure this is your job. It's a spec change. It needs additional consideration. I just wanted to mention it here.